### ISSUE TITLE
Request For New OAuth plugin for MojioCarConnect

### ISSUE THREAD

**Issue by anveshnara** [nraboy/ng-cordova-oauth#207]
would be fantastic, see https://developer.moj.io/reference/documentation for detalils

**Comment by nraboy** [comment]
It doesn't look like it uses Oauth.  Can you prove it does?

**Comment by anveshnara** [comment]
Yes, mojio-js-cordova-example-oauth2 already there is example in Github check it out.

**Comment by nraboy** [comment]
Link to the official oauth documentation please.  We are not going to scavenge GitHub to find the information we need.

Regards,

**Comment by anveshnara** [comment]
hey I code entire MojioOAuth Integeration how can I merge the code ?

**Comment by nraboy** [comment]
Make a fork of ng-cordova-oauth and clone the fork to your computer.  The clone will have the `master` branch checked out.  You must checkout the `development` branch like so:

```
git checkout development
```

When the development branch showing, copy your changes in and then commit them.  Push your changes to your remote fork on GitHub.  Then create a pull request to merge your development branch with my development branch.

This might help too:

https://blog.nraboy.com/2015/02/contributing-git-repository-github/

Best,

**Comment by anveshnara** [comment]
when I commit changes I got 2 errors 
error: pathspec 'Mojio' did not match any file(s) known to git.
error: pathspec 'OAUTH' did not match any file(s) known to git.

**Comment by nraboy** [comment]
Sounds like you didn't merge something correctly.  I suggestion reading up on Git.

Best,
